src/main.rs

```diff
@@ -35,7 +35,7 @@ enum Command {
     /// Initialize the HD wallet. If it is already initialized this will remove the old one.
     Init {
         #[clap(long)]
-        phrase: Option<String>,
+        import: bool,
```

Comment:
I wonder if we should instead create a dedicated `forc-wallet import` command for this?

Reply:
+1 I think it should be a different command.

Reply:
Opened #37 to track that